packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -288,3 +288,30 @@ export interface ICreateChildDetails {
     /** Sequence number of latest known change to the node */
     changeSequenceNumber: number;
 }
+
+export interface ISubtreeInfo {
+    /** Tree to use to find children subtrees */
+    childrenTree: ISnapshotTree,
+    /** Additional path part where children are isolated */
+    childrenPathPart: string | undefined,
```

Comment:
I probably did not stair too long at the code to better understand it, but it is not clear to me from glancing what the actual meaning of childrenPathPart and how callers of API below use it. Maybe it's just me (and not having enough focus), but would it be helpful to have a bit more in comment on when it's undefined, and when it's not, what it means and how this data is expected to be used?